mikeio/dfsu.py

```diff
@@ -1082,11 +1082,12 @@ def plot(self, z=None, elements=None, label=None, cmap=None, vmin=None, vmax=Non
             else:
                 # must be contourf plot then
                 levels = np.linspace(vmin, vmax, n_levels)
+                ax.tripcolor(triang, zn, edgecolors='face', vmin=vmin, vmax=vmax, cmap=cmap)
                 tr_fig = ax.tricontourf(triang, zn, levels=levels, cmap=cmap)
-                # if plot_type == 'contour_lines':
-                #     ax.tricontour(triang, zn, levels=levels,
-                #             colors=['0.5'],
-                #             linewidths=[0.5])
+                if plot_type == 'contour_lines':
```

Comment:
@mohm-dhi, do you know how we could make a non-filled contour plot (which I guess is relevant expectation if a user requests a contour plot)

![image](https://user-images.githubusercontent.com/34088801/93066960-e3696680-f67a-11ea-9c2b-70ef6c8b9efb.png)

Reply:
well, tricontour creates the lines and tricontourf make it filled contour. So if we want to have just the lines we should not have tricontourf